TabZilla/tabzilla_utils.py

```diff
@@ -226,40 +226,42 @@ def cross_validation(model: BaseModel, dataset: TabularDataset, time_limit: int,
             verbose=False,
             scaler=scaler,
             one_hot_encode=False,
+            args=args,
         )
         X_train, y_train = processed_data["data_train"]
         X_val, y_val = processed_data["data_val"]
         X_test, y_test = processed_data["data_test"]
-
         # Create a new unfitted version of the model
         curr_model = model.clone()
 
         # Train model
         timers["train"].start()
         # loss history can be saved if needed
+        print("Fitting model, iteration ", i, " of ", len(dataset.split_indeces))
         loss_history, val_loss_history = curr_model.fit(
             X_train,
             y_train,
             X_val,
             y_val,
         )
+        print("Done fitting model, history is: ", loss_history, val_loss_history)
```

Comment:
this could produce a lot of output, I would rather not include this print statement